packages/mobile/src/backup/BackupComplete.tsx

```diff
@@ -48,19 +36,16 @@ class BackupComplete extends React.Component<Props> {
         throw new Error('Backup complete screen should not be reachable without completing backup')
       }
     }, 2000)
-  }
+  }, [])
```

Comment:
nit: i think we should also return a `clearTimer` function here to cleanup the `useEffect` after component unmount? this was missing in the previous implementation but for whatever reason if this component is unmounted before 2 seconds then this setTimeout will be hanging out :)

Reply:
Should be fixed in https://github.com/valora-inc/wallet/pull/1676/commits/ddc035bd8314453a01fde0f89275674f373f3a86!